4 bug fix PRs

hdunham opened the pull request:
See PRs:
- #371
- https://github.com/NREL/REopt.jl/pull/375 
- #346
- #325

Bill-Becker commented:
@hdunham added a hot-fix to this PR FYI. Want to change the reviewer to me to expedite?

Bill-Becker commented:
@hdunham I have some very small but important fixes that I'd like to merge and get into the API ASAP. Let me know if you want to merge any more of your open PRs before we merge this.

hdunham commented:
> @hdunham I have some very small but important fixes that I'd like to merge and get into the API ASAP. Let me know if you want to merge any more of your open PRs before we merge this.

Just addressed your comments on the fuel slope/intercept PR. I believe that's the last one that could be added, once tests rerun.
